Libraries/Image/Image.ios.js

```diff
@@ -66,6 +66,11 @@ var Image = React.createClass({
       uri: PropTypes.string,
     }),
     /**
+     * Force display of the ALAsset in thumbnail format 
+     * This property reduce the memory size and only work with ALAsset
+     */
+    assetThumbnail: PropTypes.bool,
```

Comment:
I think the `assetThumbnail` prop should be moved into the `source` object - `{{uri: '...', thumbnail: true}}`